opentech/apply/projects/templates/application_projects/project_detail.html

```diff
@@ -166,11 +166,26 @@ <h4 class="modal__header-bar">Request Payment</h4>
 
                 </div>
 
+                {% if object.is_in_progress %}
+                <div class="sidebar__inner">
+                    <h5>Supporting Documents</h5>
+
+                    <p><a href="{{ object.submission.get_absolute_url }}">Proposal</a></p>
+
+                    {% if request.user.is_apply_staff %}
+                    <p><a href="#">Approval form</a></p>
+                    <p><a href="#">Supporting documents</a></p>
+                    {% endif %}
+
+                    <p><a href="{{ object.contracts.first.file.url }}">Draft contract</a></p>
```

Comment:
> Template updating to reflect the design (contracts subheading within supporting docs sidebar)

As per the spec (Page Variations -> In Progress -> Design (opens Page 10)) the designs show "Draft contract" in the sidebar.

Could we do a better job at finding a draft contract? In hindsight, yes, but this is here because the ticket and design say it should be, if either of those are incorrect then we should fix those.

Reply:
lets remove it - we've promoted contracts to its own block and are showing that at all times so the designs are no longer relevant.

Reply:
Fixed in 09d602c3